src/bin/tattoo.ts

```diff
@@ -13,14 +13,19 @@
  * http://polymer.github.io/PATENTS.txt
  */
 
-import {CliOptions, getCommandLineOptions, showCliHelp, loadConfigFileOptions, mergeConfigFileOptions, ensureGitHubToken} from '../cli';
+import {CliOptions, getCommandLineOptions, getCliHelp, getVersion, loadConfigFileOptions, mergeConfigFileOptions, ensureGitHubToken} from '../cli';
 import {Runner, RunnerOptions} from '../runner';
 
 async function main() {
   console.time('tattoo');
   try {
     const cliOptions: CliOptions = getCommandLineOptions();
-    showCliHelp(cliOptions);
+    if (cliOptions.version) {
+      return console.log(getVersion());
```

Comment:
ts nit: returning console.log here will anger typescript (esp. when we eventually turn noImplicitAny on). Split this into two lines (`console.log()` and then `return;`)